nixos/modules/services/misc/mollysocket.nix

```diff
@@ -0,0 +1,133 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
```

Comment:
Late to the party, but there is a consensus against top-level with statements. Should be replaced with `inherit (attrset) value` instead.